<launch>
  <rosparam command="load" file="$(find uav_nav)/config/uav_nav/uav_nav_params.yaml"/>
  <node name="depth_gen_front" pkg="uav_nav" type="depth_generation_front" output="screen"/>
  <node name="depth_gen_left" pkg="uav_nav" type="depth_generation_left" output="screen"/>
  <node name="depth_gen_right" pkg="uav_nav" type="depth_generation_right" output="screen"/>
  <node name="vfh_service_server" pkg="uav_nav" type="vfh_server" output="screen"/>
  <node name="vfh_plotter" pkg="uav_nav" type="vfh_plot" output="screen"/>
  <!--node name="safety_interrupt" pkg="uav_nav" type="safety_interrupt" output="screen"/-->
</launch>
